### PR TITLE
support fuse kill_priv_v2 feature

### PR DESCRIPTION
**Pull request by bergwolf**
So that we can make kernel skip querying file system about a file's xattr during write, which would kill wirte performance.

### CODE REVIEW

File: src/passthrough/async_io.rs [hunk 1]
```diff
@@ -97,8 +97,9 @@ impl<D: AsyncDrive> PassthroughFs<D> {
         ctx: &Context,
         inode: Inode,
         flags: u32,
-        fuse_flags: u32,
+        _fuse_flags: u32,
     ) -> io::Result<(Option<Handle>, OpenOptions)> {
+        // FIXME: handle FOPEN_IN_KILL_SUIDGID properly
```
Comment (jiangliu) [inline src/passthrough/async_io.rs:102]: Do we need to enhance the io_uring subsystem to support this feature?

**Comment by jiangliu** [comment]
Could you please also help to modify the CHANGELOG.md file?

File: src/passthrough/async_io.rs (hunk 1 above)
Comment (bergwolf) [reply src/passthrough/async_io.rs:102]: yup, async IO, we are not sure if it is still the same thread when reverting the thread cap changes in IO finish callback.

**Comment by jiangliu** [comment]
Replaced by #35

**Pull request closed without merging**